src/Time/Tags.hpp

```diff
@@ -85,6 +86,7 @@ namespace OptionTags {
 struct FinalTime {
```

Comment:
Just a heads-up: This is removed in #1520 so whichever PR is merged second will have some conflicts to resolve.